src/parameterized_problem.cpp

```diff
@@ -652,16 +676,25 @@ LinElastDispLattice::LinElastDispLattice()
     : LinElastProblem()
 {
    // pointer to static function.
-   bdr_type.SetSize(2);
-   battr.SetSize(2);
-   vector_bdr_ptr.SetSize(2);
-   for (size_t i = 0; i < vector_bdr_ptr.Size(); i++)
+   bdr_type.SetSize(3);
+   battr.SetSize(3);
+   vector_bdr_ptr.SetSize(3);
+  for (size_t i = 0; i < vector_bdr_ptr.Size(); i++)
+      {
+if (i<2)
```

Comment:
Just for readability: can we just loop up to `2`, and write the last loop separately?